KarhooUISDK/UIComponents/LoyaltyView/KarhooLoyaltyView.swift

```diff
@@ -277,4 +296,9 @@ extension KarhooLoyaltyView: LoyaltyView {
     func hasError() -> Bool {
         return presenter?.hasError() ?? false
     }
+    
+    private func updateBalanceView(mode: LoyaltyBalanceMode) {
```

Comment:
I have a problem with the name of this method. In my opinion, it primarily sets `balanceView` mode, updating the value (sort of) by the way.

Reply:
Update the name to `refreshBalanceView(with mode: LoyaltyBalanceMode)` since it's main purpose is to refresh the UI based on the given mode